backend/supervisor.d/services.conf

```diff
@@ -71,8 +71,8 @@ stdout_logfile_maxbytes=0
 
 [program:database]
 command=/usr/local/bin/python -c "import %(program_name)s; %(program_name)s.run()"
-process_name=%(program_name)s
-numprocs=1
+process_name=%(program_name)s_%(process_num)s
+numprocs=10
```

Comment:
Should this be 10 by default?

Reply:
we can change it to one but we need to somehow inform the user when it is needed to add more DB clients..

Reply:
Sounds good. Resolving comment.